benchmark/benchmark_control.py

```diff
@@ -0,0 +1,49 @@
+import sys
+
+from eliot import to_file
+
+from twisted.internet.task import react
+from twisted.python.usage import Options, UsageError
+
+from benchmark_driver import driver
+from benchmark_scenarios import supported_scenarios, default_scenario
+from benchmark_operations import supported_operations, default_operation
+from benchmark_measurements import supported_measurements, default_measurement
```

Comment:
Any particular reason for all of these modules to be prefixed w/ `benchmark_`? Since this is a package already, why not call them `_driver', `_scenarios`, etc.?